### ISSUE TITLE
Embedded MKV S_TEXT/UTF8 subtitles don't render correctly

### ISSUE THREAD

**Issue by Wuzado** [jellyfin/jellyfin#2841]
**Describe the bug**
When watching a movie with multiple dialogue lines on screen at the same time, only one dialogue line is rendered, often at a wrong place on screen. When watching on VLC, it's rendered correctly though.

**System (please complete the following information):**
 - OS: Windows 10
 - Virtualization: None
 - Clients: Browser
 - Browser: Firefox 75 on Windows 10 (same PC)
 - Jellyfin Version: tested on stable 10.5.3 and nightly 20200408
 - Playback: WebUI?
 - Installed Plugins: N/A
 - Reverse Proxy: None
 - Base URL: localhost:8096
 - Networking: hosted on LAN, played back at the same PC
 - Storage: local

**To Reproduce**
1. Watch a movie with multiple dialogue lines embedded into MKV container.

**Expected behavior**
VLC:
![VLC](https://user-images.githubusercontent.com/23276409/79069263-2a6abe80-7ccd-11ea-8950-8184c0a46b12.PNG)

**Actual behavior**
![jellyfin](https://user-images.githubusercontent.com/23276409/79069272-3c4c6180-7ccd-11ea-8644-2ee7ce646d04.PNG)

**Comment by Artiume** [comment]
@JustAMan @sparky8251 found some more subtitles issues.

**Comment by JustAMan** [comment]
Indeed. Sounds a bit similar to https://github.com/jellyfin/jellyfin-web/issues/795 but is actually not.

@Wuzado does it work in Chrome? Can you provide us with sample `.SRT` file that exhibits the behaviour?

Also I'm moving it to jf-web.
